.env

```diff
@@ -10,3 +10,6 @@ VUE_APP_BACKEND_UPDATE_USER_CONFIG=/users/update_config
 VUE_APP_BACKEND_LOGIN_USER=/users/login
 VUE_APP_I18N_LOCALE=hi
 VUE_APP_I18N_FALLBACK_LOCALE=en
+
+VUE_APP_BACKEND_NEW=http://0.0.0.0:8001/api/v1/
+VUE_APP_BACKEND_PLIO_NEW=plios/
```

Comment:
Temporary variables for now, will replace the old ones when as we go on

Reply:
need to overwrite older one